i18n/extract.py

```diff
@@ -56,7 +56,10 @@ def rename_source_file(self, src, dst):
         """
         Rename a file in the source directory.
         """
-        os.rename(self.source_msgs_dir.joinpath(src), self.source_msgs_dir.joinpath(dst))
+        try:
+            os.rename(self.source_msgs_dir.joinpath(src), self.source_msgs_dir.joinpath(dst))
+        except OSError:
+            pass
```

Comment:
I assume there's a good reason to ignore the error? A comment here will soothe readers' frazzled nerves :)

Reply:
The initial reason for this was to help out as I tried to get the files that get left in a changed state after testing under control. Underneath it are some problems with extract getting settings when under test. I've spent too much time trying to get them to work in this task so I'm going to re-skip them in the next commit and add a task to come back and fix that up.